src/components/Scale.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react'
+import { useMapInstance } from '@datapunt/react-maps'
+import { Control, control } from 'leaflet'
+
+import ScaleOptions = Control.ScaleOptions
+
+type Props = {
+  options?: ScaleOptions
```

Comment:
Should `options` not be required?

Reply:
nope, https://leafletjs.com/reference-1.6.0.html#control-scale